code/game/jobs/jobs.dm

```diff
@@ -65,12 +65,19 @@ var/list/command_positions = list(
 	"Head of Security",
 	"Chief Engineer",
 	"Research Director",
-	"Chief Medical Officer"
+	"Chief Medical Officer",
+	"Acting Captain",
+	"Acting Head of Personnel",
+	"Acting Head of Security",
+	"Acting Chief Engineer",
+	"Acting Research Director",
+	"Acting Chief Medical Officer"

```

Comment:
I considered doing this, but for encouragement of people to use the fill title decided against it. If the maintainers would like this to be included, I'll add it in, but I'm not a fan of people using the abbreviations.